test/ui-testing/tags-crud.js

```diff
@@ -0,0 +1,131 @@
+/* global describe, it, before, after, Nightmare */
+const AgreementCRUD = require('./agreement-crud');
+
+module.exports.test = (uiTestCtx) => {
+  const number = Math.round(Math.random() * 100000);
+  const testTag = `test${Math.floor(Math.random() * 10000)}`;
+  const agreement = {
+    name: `Linked License Agreement #${number}`,
```

Comment:
Call it something like `Tags Agreement #${number}` instead. Helps to distinguish the agreements created after the fact.